CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.14 FATAL_ERROR)
+PROJECT(Libtcod VERSION 1.61.2 LANGUAGES CXX C)
+
+SET(PATH_TO_DATA ${CMAKE_CURRENT_SOURCE_DIR}/data/)
+SET(PATH_TO_TERMINAL_PNG ${CMAKE_CURRENT_SOURCE_DIR}/terminal.png)
```

Comment:
These variables had to be established here, because otherwise, their definition inside another file required the use of small tricks that were not well seen. I will be honest again, this Pull was done in a hasty way and skipping all the quality controls I work with, I did it this way because I wanted to determine if it was my machine that was failing (which has already been refuted).